GVRf/Extensions/3DCursor/IODevices/gvrmobileiodevice/src/main/java/com/example/gvrmobileiodevice/ConnectServer.java

```diff
@@ -0,0 +1,85 @@
+package com.example.gvrmobileiodevice;
```

Comment:
can use com.gearvrf.io.gvrmobileiodevice as package name, to keep inconsistent with the gearwear io device.